src/routes/game/GameView.vue

```diff
@@ -505,26 +505,26 @@ export default {
       return this.gameStore;
```

Comment:
Let's also remove the `game()` computed property on line 504 here so that we can clearly see any dangling references (as they'll error).

You'll also need to replace the `game` references in the template code above e.g. line 128 and 315. I recommend searching for `game.` in this file